resources/knative-eventing/charts/event-mesh-dashboard/templates/event-mesh-pods-dashboard.yaml

```diff
@@ -1,551 +1,551 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-    name: {{ template "event-mesh-dashboard.fullname" . }}-pods
-    namespace: kyma-system
-    labels:
-        grafana_dashboard: "1"
-{{ include "event-mesh-dashboard.labels.standard" . | indent 8 }}
+  name: {{ template "event-mesh-dashboard.fullname" . }}-pods
+  namespace: kyma-system
+  labels:
+    grafana_dashboard: "1"
+{{ include "event-mesh-dashboard.labels.standard" . | indent 4 }}
 data:
-    event-mesh-pods-dashboard.json: |-
-        {
-        "annotations": {
```

Comment:
From the diff it may look like I changed that entire file, but all I did was pass the data to `jq` in order to fix the broken indentation.